tests/macros/src/main/scala/scala/macros/tests/scaladays/TestMacros.scala

```diff
@@ -0,0 +1,10 @@
+package scala.macros.tests.scaladays
+
+import scala.macros._
+
+import scala.language.experimental.macros
```

Comment:
I've added `-language:experimental.macros` to scalacOptions to avoid the need for this line.  The scalac-plugin SyntaxAnalyzer inserts `import scala.language.exprimental.macros` but it doesn't seem to be picked up by   scalac for all macros, only the Serialize macro.